backend/service/authn/oidc.go

```diff
@@ -133,6 +133,23 @@ func (p *OIDCProvider) Exchange(ctx context.Context, code string) (*oauth2.Token
 		}
 	}
 
+	return p.signNewToken(ctx, claims)
+}
+
+func (p *OIDCProvider) CreateToken(ctx context.Context, subject string, expiry time.Duration) (*oauth2.Token, error) {
+	claims := &Claims{
+		StandardClaims: &jwt.StandardClaims{
+			ExpiresAt: time.Now().Add(expiry).Unix(),
+			IssuedAt:  time.Now().Unix(),
+			Issuer:    clutchProvider,
+			Subject:   subject,
```

Comment:
we should enforce some prefix for the subject so you can't reasonably override a user, and so that you can tell what type of user it is more easily in logs.

for example we have `system:anonymous`, maybe `service:<subject>`?